apis/Google.Cloud.Spanner.Data/docs/index.md

```diff
@@ -254,3 +254,34 @@ If you want to diagnose session management issues,
 the statistics for the session pool associated with the connection:
 
 {{sample:SpannerConnection.GetSessionPoolSegmentStatistics}}
+
+## RpcException from SpannerException
+
+The `RpcException` property of `SpannerException` returns the underlying
+`RpcException` which caused the `SpannerException`. The returned 
+value may be null if the `SpannerException` was not caused by an `RpcException`.
+
+`ErrorInfo` in [RpcException](../../../docs/reference/help/troubleshooting#how-can-i-get-errorinfo-from-rpcexception) can 
```

Comment:
This doesn't appear to be the right URL and I guess this link will be broken.
I am not sure of the convention we follow in docs and if url must be a relative url or the absolute web url. Please check with Jon. I do see few absolute urls as well. 
But if we are to use relative url - it should have been (../../../docs/devsite-help/troubleshooting.md#how-can-i-get-errorinfo-from-rpcexception) or the absolute url (if its allowed) should be [RpcException](https://cloud.google.com/dotnet/docs/reference/help/troubleshooting#how-can-i-get-errorinfo-from-rpcexception)

Reply:
Thanks Rishabh, updated the relative url as per your suggestion. @jskeet Please let us know that the relative url seems fine to you or we should use an absolute url pointing to RpcExceptions help page?